mappings/net/minecraft/client/player/EntityPlayerClient.mapping

```diff
@@ -5,6 +5,7 @@ CLASS none/boe net/minecraft/client/player/EntityPlayerClient
 	METHOD a swingHand (Lnone/qz;)V
 		ARG 0 hand
 	METHOD c getBlockPos ()Lnone/cn;
+	METHOD c setCurrentHand (Lnone/qz;)V
```

Comment:
This method is inherited from [EntitySomethingBase](https://github.com/Darkhax-Forked/pomf/blob/74504345865945d8a0f2c4af4162e65e5eb6ac20/mappings/net/minecraft/sortme/EntitySomethingBase.mapping#L10).